torch/quantization/_numeric_suite.py

```diff
@@ -337,7 +359,10 @@ def compare_model_stub(
             quantized module and its float shadow module
     """
     prepare_model_with_stubs(float_model, q_model, module_swap_list, Logger)
-    q_model(data)
+    if type(data) is tuple:
```

Comment:
We should always pass all the inputs to the model. Why is there a need to have a check here?

Reply:
This is because LSTM module takes two inputs not tuple, so if data is tuple we need to unpack it into two inputs by doing *data.

Reply:
I am still wondering if something simpler is possible: Here is an example:

https://www.internalfb.com/intern/anp/view/?id=285847